src/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -123,23 +122,17 @@ export const CollectionApp: React.FC<CollectionAppProps> = props => {
           counts={collection.artworksConnection?.counts as Counts}
         />
 
-        {/* HOTFIX FIXME: This rail was causing an error if included in SSR render
-              pass and so it was deferred to the client.
-
-              See: https://github.com/artsy/force/pull/6137
-          */}
-        {collection.linkedCollections.length === 0 &&
-          typeof window !== "undefined" && (
```

Comment:
This also would have been causing a server-side rendering mismatch.

Reply:
I remember this -- for whatever reason which made zero sense at all and took ages to track down -- would cause a full on crash somewhere in the router sometimes due to some kind of subtle race condition. This was way long ago and could definitively have been fixed somewhere along the way by us just moving everything into a proper SPA and so on and so forth